index.html

```diff
@@ -222,7 +222,7 @@ <h2 class="view">I'm always<br>
             <hr>
             <input type="text" class="input" placeholder="Email address">
             <hr>
-            <input type="text" class="box" placeholder="Enter text here">
+            <textarea class="box" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] Following the project requirements, All input fields should have the required attribute to activate the validation on submit. Kindly add it 😃.
- [ ] Message input also should have `maxlength="500"` attribute.